emcc.py

```diff
@@ -2545,10 +2545,6 @@ def phase_linker_setup(options, state, newargs):
       # denominator and disable bulk memory altogether for Node.js < 18 or to
       # special-case this situation here. The former would be limiting for
       # wasm32 users, so instead we do the latter:
-      logger.warning(
-        "Disabling bulk memory because it doesn't work correctly with wasm64 in Node.js < 18.\n"
-        "Set MIN_NODE_VERSION to 180000 or above to enable it."
-      )
       settings.BULK_MEMORY = 0
```

Comment:
I agree a warning is a lot, but maybe it could be downgraded to appear in verbose logging modes? I can imagine it might help figure something out that way, some day.